src/utils/ParseRouteUtils.js

```diff
@@ -183,6 +183,12 @@ async function condenseRoute(
       if (startTime < departureTimeNowMs - ONE_MIN_IN_MS - delayMs) {
         return null;
       }
```

Comment:
curious what is the value of `delayMs` here?

Reply:
i think it's literally the delay, so that we make sure that users can still catch buses that are late. but i feel like delay should actually be added onto departureTimeNow ?

Reply:
agh ive been comparing between tranist prod and transit beta (which is running my change), and dev just seems to make a lot more sense so idk

Reply:
yeah I feel like the issue we should be looking into is how these routes make it past this check. I would imagine the delayMs isnt that large?

Reply:
@kevinchan159 if i remember correctly from when I checked the blame, you introduced delayMs into the codebase!